src/directives/threema_action.ts

```diff
@@ -76,19 +92,38 @@ export default [
                     for (let i = 0; i < el[0].childNodes.length; i++) {
                         const node: HTMLElement = el[0].childNodes[i];
 
-                        if (node.nodeType === Node.ELEMENT_NODE
-                            && node.tagName.toLowerCase() === 'a') {
-
-                            const link = (node as HTMLElement).innerText;
-                            if (link !== undefined && link.toLowerCase().startsWith('threema://')) {
-                                const matches = (/\bthreema:\/\/([a-z]+)\?([^\s]+)\b/gi).exec(link);
-                                if (matches !== null) {
-                                    const handler = getThreemaActionHandler(matches[1]);
-                                    const params = uriService.parseQueryParams(matches[2]);
-                                    if (handler !== null && params !== null) {
-                                        node.addEventListener('click', handler(params));
+                        if (node.nodeType === Node.ELEMENT_NODE) {
+                            switch ( node.tagName.toLowerCase()) {
+                                case 'a':
+                                    const link = (node as HTMLElement).innerText;
+                                    if (link !== undefined && link.toLowerCase().startsWith('threema://')) {
+                                        const matches = (/\bthreema:\/\/([a-z]+)\?([^\s]+)\b/gi).exec(link);
+                                        if (matches !== null) {
+                                            const handler = getThreemaActionHandler(matches[1]);
+                                            const params = uriService.parseQueryParams(matches[2]);
+                                            if (handler !== null && params !== null) {
+                                                node.addEventListener('click', handler(params));
+                                            }
+                                        }
+                                    }
+                                    break;
+                                case 'span':
+                                    // supprt only id mentions (not all or me)
+                                    const mentionCssClass = 'mention id';
+                                    const cls = node.getAttribute('class');
+                                    if (cls.substr(0, mentionCssClass.length) === mentionCssClass) {
+                                        // Hack to extract the identity from class name
+                                        const identity = cls.substring(mentionCssClass.length).trim();
```

Comment:
Instead of using the css class hack, you could use a `data-mention-id` attribute (see https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes). Would make the code much cleaner :)

Reply:
The problem is the `ng-bind-html` of angularJS... the directive removes all arguments from the DOMElement (except `class`)

Reply:
I see :(

Reply:
I see :( Then maybe add that information to the comment?

Reply:
+1